_pages/documentation/gem5-stdlib/7-using-local-data-sources.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: documentation
+title: Using local Data sources in gem5
+parent: gem5-standard-library
+doc: gem5 documentation
+permalink: /documentation/gem5-stdlib/using-local-resources
+author: Harshil Patel
+---
+
+gem5 supports using local data sources in the form of a MongoDB Atlas and JSON datasource. To utilize data sources other than the main gem5 resources database, you will need to override the gem5-resources-config.
+
+MongoDB Config Format:
```

Comment:
There are a few things that need more explanation.
- Each field of the MongoDB format and the raw JSON file format should have a brief description.
- Since the MongoDB JSON and the raw JSON files have different amount of information, we'd need to explain why or when to use what.